.github/workflows/update-bpmn-visualization-version.yml

```diff
@@ -15,33 +15,33 @@ jobs:
       VERSION: ${{ github.event.client_payload.version || github.event.inputs.version }}
     steps:
       - uses: actions/checkout@v3
-      - name: Get old bpmn-visualization version
+      - name: Get the old version of bpmn-visualization
         id: lookupOldBPMNVisuVersion
         uses: mikefarah/yq@v4.30.5
         with:
-          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualization.yaml
+          cmd: yq '.dependencies[] | select(.name == "bpmn-visualization") | .version' inst/htmlwidgets/bpmnVisualizationR.yaml
       - run: echo "OLD_VERSION=${{ steps.lookupOldBPMNVisuVersion.outputs.result }}" >> $GITHUB_ENV
-      - name: Delete old bpmn-visualization js lib file
+      - name: Delete the old file of bpmn-visualization
```

Comment:
Is it simpler?
```suggestion
      - name: Delete the old bpmn-visualization
```

Reply:
"Delete the old bpmn-visualization file" ?

Reply:
In the `Download` step, there is no mention of a `file` in the title, although the same file is being handled. 
I find that there is a problem of consistency.

Reply:
as you wish